sdk/src/main/java/io/radar/sdk/model/RadarUser.kt

```diff
@@ -119,9 +119,20 @@ class RadarUser(
     /**
      * The user's current trip.
      */
-    val trip: RadarTrip?
+    val trip: RadarTrip?,
+
+    /**
+     * A boolean indicating whether the user's location is being mocked, such as in a simulation. May be `null` if
```

Comment:
think this should say "May be `false`" like we have for `proxy`

Reply:
No - see above